st2api/st2api/controllers/v1/actionexecutions.py

```diff
@@ -126,7 +128,7 @@ def _handle_schedule_execution(self, liveaction_api):
             LOG.exception('Unable to execute action. Unexpected error encountered.')
             abort(http_client.INTERNAL_SERVER_ERROR, str(e))
 
-    def _schedule_execution(self, liveaction, user=None):
+    def _schedule_execution(self, liveaction, user=None, **kwargs):
```

Comment:
Not a fan of this kwargs approach since it's not explicit which arguments can be present here.

I know this is not your fault because previously code did a very similar thing, it just didn't use `kwargs`, but it would still be better if we can come up with some more explicit and obvious approach.